src/intersection-observer-polyfill.js

```diff
@@ -273,6 +289,25 @@ export class IntersectionObserverPolyfill {
       }
     }
 
+
+    // Add a mutation observer to tick ourself
+    const ampdoc = Services.ampdoc(element);
+    if (ampdoc.win.MutationObserver && !this.mutationObserver_) {
+      this.viewport_ = Services.viewportForDoc(element);
+      this.mutationPass_ = new Pass(ampdoc.win, () => {
+        if (this.viewport_) {
+          this.tick(this.viewport_.getRect());
+        }
+      });
+      this.mutationObserver_ = new ampdoc.win.MutationObserver(
+          this.handleMutationObserverNotification_.bind(this)
+      );
+      this.mutationObserver_.observe(ampdoc.getRootNode(), {
+        attributes: true,
```

Comment:
do we need these properties preserved (unmangled)?

Reply:
Discussed offline: "it should get preserved if it knows its a mutation observer instance"